cpp/src/NVText.cu

```diff
@@ -31,6 +31,25 @@
 
 typedef custring_view** custring_view_array;
 
+void* device_alloc(size_t bytes, cudaStream_t sid)
```

Comment:
Instead of redefining this function in several places, how about instead putting it into it's own header file and including it where needed?

Reply:
Yes, added it to the util.h for this.